RELEASE/scripts/autoscend/auto_buff.ash

```diff
@@ -494,6 +501,7 @@ boolean buffMaintain(effect buff, int mp_min, int casts, int turns, boolean spec
 	case $effect[Merry Smithsness]:				useItem = $item[Flaskfull of Hollow];			break;
 	case $effect[Mind Vision]:					useSkill = $skill[Intracranial Eye];			break;
 	case $effect[Ministrations in the Dark]:	useItem = $item[EMD Holo-Record];				break;
+	case $effect[Misplaced Rage]:				useItem = $item[angry agate];					break;
```

Comment:
Add to `auto_MaxMLToCap()`. This function is sorted from big ML buffs to small ones. Not obvious where to put this one since it is 3x level. I'll go with wherever you think makes the most sense